### ISSUE TITLE
Support for conda package is quietly removed

### ISSUE THREAD

**Issue by function2-llx** [facebookresearch/xformers#1183]
I have noticed some recent commits seemingly in favor of removing support for conda packages (6e10bd21ac6fc878657b24684723ccd05e41d385 and 89fc9947921cdae8f75723b43ea7d5a000371615). These commits link to internal Meta repository and I am unable to inspect further details, and I can only read from commit messages. The commit message for 6e10bd21ac6fc878657b24684723ccd05e41d385 is rather surprising:

```
Remove build of conda binaries, as PyTorch no longer publishes on conda
```

Really? At least for now, conda is still the recommended package manager according to the official PyTorch tutorial([direct link](https://pytorch.org/get-started/locally/#linux-package-manager); [archive](https://web.archive.org/web/20241221100958/https://pytorch.org/get-started/locally#linux-package-manager)):
```
To install the PyTorch binaries, you will need to use one of two supported package managers: Anaconda or pip.
Anaconda is the recommended package manager as it will provide you all of the PyTorch dependencies in one, sandboxed install, including Python.
```

I did a preliminary search in the PyTorch repo but cannot find any hint.

Could you please provide some clarification on this topic?

**Comment by lw** [comment]
PyTorch has stopped publishing conda builds to its nightly channels, and the next stable release won't provide conda packages either. Since xFormers targets only the latest PyTorch, we have followed their direction, since we depend on PyTorch.

This does not mean that xFormers stopped working with the latest stable PyTorch, since we already published a conda package that is compatible with the latest PyTorch conda packages.
